From fee5ca8fb6adb1705fb0bc26c10264c8169af2c9 Mon Sep 17 00:00:00 2001
From: Gaurav Vohra <gauravx.vohra@intel.com>
Date: Fri, 8 Feb 2019 19:01:29 +0800
Subject: [PATCH 107/120] ASoC: Intel: Skylake: Fixed compilation failure for
 CNL

	Due to changes in skl_base_fmt from ASoC: Intel: Skylake: Correct skl_base_cfg declaration
	compilation issue is faced since there is no cps member, updating the cps values to cpc*1000

Change-Id: Ida7e1e8850d62b0152110e9c550210e6a673c847
Signed-off-by: Gaurav Vohra <gauravx.vohra@intel.com>
---
 sound/soc/intel/skylake/skl-messages.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-messages.c b/sound/soc/intel/skylake/skl-messages.c
index a48262d..dad000a 100644
--- a/sound/soc/intel/skylake/skl-messages.c
+++ b/sound/soc/intel/skylake/skl-messages.c
@@ -467,7 +467,7 @@ static int cnl_sdw_bra_pipe_cfg_pb(struct skl_sst *ctx,
 		goto error_delete_pipeline;
 	}
 
-	host_cpr_cfg->module->resources[0].cps = 100000;
+	host_cpr_cfg->module->resources[0].cpc = 100000000;
 	host_cpr_cfg->module->resources[0].is_pages = 0;
 	host_cpr_cfg->module->resources[0].ibs = 384;
 	host_cpr_cfg->module->resources[0].obs = 384;
@@ -740,7 +740,7 @@ static int cnl_sdw_bra_pipe_cfg_cp(struct skl_sst *ctx,
 		goto error_delete_pipeline;
 	}
 
-	link_cpr_cfg->module->resources[0].cps = 100000;
+	link_cpr_cfg->module->resources[0].cpc = 100000000;
 	link_cpr_cfg->module->resources[0].is_pages = 0;
 	link_cpr_cfg->module->resources[0].ibs = 1152;
 	link_cpr_cfg->module->resources[0].obs = 1152;
-- 
1.9.1

